cli/Application/Command/Install/Install.php

```diff
@@ -68,7 +68,7 @@ public function execute()
 			if (!$this->getApplication()->input->get('reinstall'))
 			{
 				$this->out()
-					->out('<fg=black;bg=yellow>' . g11n3t('WARNING: A database has been found !!') . '</fg=black;bg=yellow>')
+					->out('<fg=black;bg=yellow>' . g11n3t('WARNING: A database has been found!') . '</fg=black;bg=yellow>')
 					->out()
 					->out(g11n3t('Do you want to reinstall ?'))
```

Comment:
@elkuku no Pleken in fr-FR as we need the space (not for fr-CA) :wink: 
I will continue working on it, step by step, and also "boost" our translators.